_templates/layout.html

```diff
@@ -31,35 +31,35 @@
   <div class="admonition-grid {% if godot_is_latest and godot_show_article_status %}admonition-grid-2x{% endif %}">
     {% if godot_is_latest %}
     <div class="admonition attention latest-notice">
-      <p class="first admonition-title">Attention</p>
+      <p class="first admonition-title">Attention: Here be dragons</p>
       <p>
-        You are reading the <code class="docutils literal notranslate"><span class="pre">latest</span></code>
-        (unstable) version of this documentation, which may document features not available
-        or compatible with Godot 3.x.
+        This is the <code class="docutils literal notranslate"><span class="pre">latest</span></code>
+        (unstable) version of this documentation, which may be outdated or
+        document features not available in or compatible with released stable versions of Godot.
```

Comment:
```suggestion
        (unstable) version of this documentation, which may document features
        not available in or compatible with released stable versions of Godot.
```

Let's drop the outdated part, as discussed.